### PR TITLE
Update tutorials and docs

### PR DESCRIPTION
**Pull request by Ghostvv**
**Proposed changes**:
- see issue #200

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

File: docs/tutorial_basics.rst [hunk 1]
```diff
@@ -78,24 +78,25 @@ So what do the different parts mean?
 | ``templates`` | template strings for the things your bot can say                                                     |
 +---------------+------------------------------------------------------------------------------------------------------+
 
-In our simple example we don't need slots, so that section doesn't appear
+In our simple example we don't need ``slots`` and ``entities``, so these sections doesn't appear
```
Comment (akeladrissner) [inline docs/tutorial_basics.rst]: these sections **don't** appear

File: docs/tutorial_basics.rst [hunk 2]
```diff
@@ -144,26 +147,32 @@ created containing the NLU model.
...
     :linenos:
     :language: md
 
-Stories begin with ``##`` and a string as an identifier. User actions start
-with an asterisk, and bot actions are specified by lines beginning with a
-dash. The end of a story is denoted by a newline. See :ref:`stories` for
-more information about the data format.
+Be aware, although it is a bit faster to write
+stories directly by hand instead of using interactive learning, special
+care needs to be taken when using slots, as they need to be properly set in the
```
Comment (akeladrissner) [inline docs/tutorial_basics.rst]: unless they're unfeaturized slots - idk if that's worth mentioning here or if its enough later in the detailed slots section

File: docs/tutorial_interactive_learning.rst [hunk 3]
```diff
@@ -137,13 +138,14 @@ example, if we type ``/greet``, we get the following prompt:
 
 This gives you all the info you should hopefully need to decide
 what the bot *should* have done. In this case, the bot chose the right
-action ('greet'), so we type ``1`` and hit enter. We continue this loop
-until the bot chooses the wrong action.
+action ('utter_greet'), so we type ``1`` and hit enter.
+Then we type ``1`` again, because 'action_listen' is correct action after greeting.
```
Comment (akeladrissner) [inline docs/tutorial_interactive_learning.rst]: is **the** correct action

File: docs/tutorial_supervised.rst [hunk 4]
```diff
@@ -46,7 +46,7 @@ project directory:
 Let's inspect the domain
 definition in ``restaurant_domain.yml``:
 
-There are two sections: ``entities`` and ``slots``, which you haven't seen in the basic tutorial.
+In addition to previous example, there are two sections: ``slots`` and ``entities``.
```
Comment (akeladrissner) [inline docs/tutorial_supervised.rst]: to **the** previous example

File: docs/tutorial_supervised.rst [hunk 5]
```diff
@@ -149,13 +147,21 @@ A Custom Dialogue Policy
...
+Check out the
 ``RestaurantPolicy`` class in ``bot.py`` for the gory details:
 
 .. literalinclude:: ../examples/restaurantbot/bot.py
    :linenos:
    :pyobject: RestaurantPolicy
 
+The parameters ``max_history_len`` and ``n_hidden``
+may be altered dependent on the task complexity and the amount of data one
+has. ``max_history_len`` is important as it is the amount of story steps the
```
Comment (akeladrissner) [inline docs/tutorial_supervised.rst]: maybe "the amount of previous story steps" - just to be clear?

File: docs/tutorial_supervised.rst [hunk 6]
```diff
@@ -196,8 +198,8 @@ To train the dialogue policy from the command line, run
 ^^^^^^^^^^^^^^^^
 
 Now we're going to glue some pieces together to create an actual bot.
-We instantiate an ``Agent``, which owns our NLU ``Interpreter``, our
-trained ``Policy``, and a ``Domain``.
+We instantiate an ``Agent``, which owns our trained ``Policy`` and a ``Domain`` from ``models/dialogue``,
```
Comment (akeladrissner) [inline docs/tutorial_supervised.rst]: trained ``Policy``, a ``Domain`` (remove the and)

File: docs/tutorial_supervised.rst [hunk 7]
```diff
@@ -236,9 +238,21 @@ e.g.:
...
+You can find a complete example of how to load an agent and chat with it on the
+command line in the restaurant bot's ``bot.py`` ``run`` method.
+To run the bot from the command line, call
+
+.. code-block:: bash
+
+    python bot.py run
+
+If the bot appears to be stuck or answers incorrectly, do not worry.
+Provided dataset is not diverse enough to handle all possible inputs,
```
Comment (akeladrissner) [inline docs/tutorial_supervised.rst]: **The** provided dataset

**Comment by akelad** [comment]
oops - somehow was logged into my other github account 😄

**Comment by tmbo** [comment]
now your secret other github account is exposed :P

**Comment by akelad** [comment]
😱 sadly there's nothing interesting on it hahaha

File: docs/tutorial_basics.rst (hunk 2 above)
Comment (Ghostvv) [reply docs/tutorial_basics.rst]: I think it is better to leave it to the Slots section. The reader has very vague idea of what a slot is right now, not speaking about the type

File: docs/tutorial_supervised.rst (hunk 6 above)
Comment (Ghostvv) [reply docs/tutorial_supervised.rst]: are you sure? sounds strange for me, without and

Comment (akelad) [reply docs/tutorial_supervised.rst]: It's a list of three things, so yeah I'd only put an "and" before the last one.

**Pull request merged**